docs/contributing/timeline-profile.md

```diff
@@ -12,29 +12,24 @@ issue.
  <img width="558" src="https://user-images.githubusercontent.com/359239/26888284-f80ed244-4b80-11e7-86cf-933c59b8f370.png">
 
  - Get the Desktop application ready to perform the problem action.
- - Select the **Timeline** tab.
+ - Select the **Performance** tab. Ensure the **Disable JavaScript samples** option is **unchecked**.
 
-<img width="1123" src="https://user-images.githubusercontent.com/359239/26887089-7c82f0cc-4b7d-11e7-9cf4-fcbd5a994e1f.png">
-
- - Ensure you have **JS Profile** checked - other items might be helpful, but
-   will add to the size of the generated timeline data.
-
-<img width="1123" src="https://user-images.githubusercontent.com/359239/26888385-3fc9157c-4b81-11e7-9083-379a37d20641.png">
+<img width="972" src="https://user-images.githubusercontent.com/359239/46921615-e9a8c100-cfd3-11e8-9a0f-eac7fda611f6.png">
 
  - Press the **Record** button on the left to start recording.
 
-<img width="1123" src="https://user-images.githubusercontent.com/359239/26887091-7c8acc3e-4b7d-11e7-9d32-961ac353f1b9.png">
+<img width="1051" src="https://user-images.githubusercontent.com/359239/46921640-3be9e200-cfd4-11e8-937b-b3ebbebaab68.png">
 
  - Perform the problem action in Desktop. Try and keep the test focused on the
    issue you're seeing.
  - Switch back to the Developer tools and press **Stop** to complete recording.
 
- <img width="1011"  src="https://user-images.githubusercontent.com/359239/26887304-0700a514-4b7e-11e7-9a1d-74fac88cecee.png">
+<img width="1050" src="https://user-images.githubusercontent.com/359239/46921658-8a977c00-cfd4-11e8-86ed-d4b6878f08c5.png">
 
- - In the graph section, right-click and select **Save timeline data**. Save the
-   JSON file somewhere you can access later.
+ - In the header, click the **Save profile...** menu item. Save the JSON file
+   somewhere you can access later.
 
- <img width="1014" alt="screen shot 2017-06-07 at 12 38 18 pm" src="https://user-images.githubusercontent.com/359239/26887386-472e9c7c-4b7e-11e7-9d39-a9b71ea3d70a.png">
+<img width="405" src="https://user-images.githubusercontent.com/359239/46921672-b450a300-cfd4-11e8-98c6-cba2b3eef1b8.png">
 
  - Compress the JSON file to reduce the file size (it could be 10MB or more
```

Comment:
Since you're already updating this file, it may be nice to explain how to compress the JSON or just say create a ZIP archive or something similar.

Reply:
`macOS` has the "Compress [file]" context menu and Windows has "Send to | Compressed (zip) file" context menu options that both create a zip file, so I'm fine with leaving it as-is. We don't need to entirely replace Google with this doc.